Instructions for Code Coverage fail on Ubuntu 16.10

** Observed Behaviour **

Having followed the instructions (COMPILING) on Ubuntu 16.10 I get the following error when attempting to run get_coverage.sh:
```
james_wilson@ubuntu:~/source/cbmc/regression$ ./get_coverage.sh 
INFO: setting up vpath build make[1]: g++: Command not found
Makefile:57: recipe for target 'library/converter' failed
make[1]: *** [library/converter] Error 127
Makefile:56: recipe for target 'ansi-c_generated_files' failed
make: *** [ansi-c_generated_files] Error 2
Failed to run 'make generated_files'
[ERROR]
ERROR: See ./coverage_SrhN5/cbmc_coverage.out for more information
```
After installing g++ using `apt-get install g++` this error goes away

**Expected Behaviour**

For the documentation to include the packages required.

**Version**

CBMC master Latest commit d999e56.

**Steps to reproduce**
1. On a clean ubuntu VM (gcloud)
2. Install packages listed in the COMPILING docs
3. `cd regression`
4. `./get_coverage.sh`

--- a/COMPILING
+++ b/COMPILING
@@ -32,7 +32,7 @@ We assume that you have a Debian/Ubuntu or Red Hat-like distribution.
    The GNU Make needs to be version 3.81 or higher.
    On Debian-like distributions, do
 
-   apt-get install g++-6 gcc flex bison make git libwww-perl patch
+   apt-get install g++ gcc flex bison make git libwww-perl patch
 
    On Red Hat/Fedora or derivates, do
 
@@ -44,13 +44,7 @@ We assume that you have a Debian/Ubuntu or Red Hat-like distribution.
 
    git clone https://github.com/diffblue/cbmc cbmc-git
 
-2) On Debian, do
-
-   cd cbmc-git/src
-   make minisat2-download
-   make CXX=g++-6
-
-   On Ubuntu, or other distributions with recent g++, do
+2) On Debian or Ubuntu, do
 
    cd cbmc-git/src
    make minisat2-download

--- a/src/jsil/Makefile
+++ b/src/jsil/Makefile
@@ -37,6 +37,8 @@ jsil_y.tab.h: jsil_y.tab.cpp
 jsil_lex.yy.cpp: scanner.l
 	$(LEX) -Pyyjsil -o$@ scanner.l
 
+generated_files: jsil_lex.yy.cpp jsil_y.tab.cpp jsil_y.tab.h
+
 # extra dependencies
 jsil_y.tab$(OBJEXT): jsil_y.tab.cpp jsil_y.tab.h
 jsil_lex.yy$(OBJEXT): jsil_y.tab.cpp jsil_lex.yy.cpp jsil_y.tab.h

